hw/src/ledger.rs

```diff
@@ -393,8 +342,17 @@ impl <'a>Wallet<'a> for Manager {
 		let devices = usb.devices();
 		let num_prev_devices = self.devices.read().len();
 
+		// Wrong state, sometimes when a ledger is connected at run-time (with no other devices connected)
```

Comment:
```suggestion
		// Sometimes when a ledger is connected at run-time with no other devices connected it will case a `disconnected` event. To work around this, ignore such spurious events and poll a couple of extra times in order to get the correct state.
```